source/features/emphasize-draft-pr-label.css

```
@@ -1,7 +1,7 @@
 .js-issue-row [aria-label='Open draft pull request'] svg {
-	stroke: #586069;
+	stroke: var(--color-text-secondary, #586069);
 	stroke-width: 1.2px;
-	color: #fff !important;
+	color: var(--color-bg-canvas, #fff) !important;
```

I think this is more appropriate, without using `important`, I don't know if it works

```suggestion
	fill: var(--color-bg-canvas, #fff);
```